components/sensor/ct_clamp.rst

```diff
@@ -38,7 +38,7 @@ Configuration variables:
 - **name** (**Required**, string): The name of the sensor.
 - **sensor** (**Required**, :ref:`config-id`): The source sensor to measure voltage values from.
 - **sample_duration** (*Optional*, :ref:`config-time`): The time duration to sample the current clamp
-  with. Higher values can increase accuracy. Defaults to ``200ms`` which would be 10 whole cycles on a 50Hz system.
+  with. Higher values can increase accuracy. Gathered data are then root mean squared (RMS). Defaults to ``200ms`` which would be 10 whole cycles on a 50Hz system.
```

Comment:
The `sample_duration` is not really the best place to put this note at IMO (and root mean square afaik is not a verb). Could we move it to the top like this?

`... convert the readings to measured single phase root mean square AC current`